sdk/src/loom2d/Display/Graphics.ls

```diff
@@ -44,29 +34,66 @@ package loom2d.display
     };
     
     [Native(managed)] 
+    /**
+     * Describe the format of some text for the Graphics class.
+     */
     public native class TextFormat
     {
+        /**
+         * Describe a font format by specifying font name, indicating size, color, and if it's bolded.
```

Comment:
Default parameters get ignored and are taken from the previous format. You can set additional properties on the object.